src/app/record/components/work-stack/work-stack.component.scss

```diff
@@ -17,3 +17,7 @@
     position: relative;
   }
 }
+
+a {
```

Comment:
Should we maybe create a global style with this?  here https://github.com/ORCID/orcid-angular/blob/dev/src/styles.scss#L59-L69

Reply:
Done!